packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -324,8 +508,33 @@ class _CheckboxPainter extends ToggleablePainter {
       ..strokeCap = StrokeCap.round;
   }
 
-  void _drawBox(Canvas canvas, Rect outer, Paint paint, BorderSide? side, bool fill) {
-    if (fill) {
+  void _drawBox(Canvas canvas, Rect outer, Paint paint, BorderSide? side) {
+    // Add dropshadow effect when unselected.
+    // The gradient colors were eyeballed from examples in the HIG docs.
```

Comment:
Can you link this and/or be more specific about versioning? I imagine someone might return to this in the future when the HIG docs have changed.